src/shared/Avatar/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { FC } from 'react';
+import Avatar from '@mui/material/Avatar';
+import { ImageAvatarsProps } from '@ts/interfaces';
+export const ImageAvatarMedium: FC<ImageAvatarsProps> = ({ avatarImg }) => {
```

Comment:
Do we need different components for different avatars?
Can we use one component instead?